packages/jest-environment/src/index.ts

```diff
@@ -44,6 +44,7 @@ export declare class JestEnvironment<Timer = unknown> {
   fakeTimersModern: ModernFakeTimers | null;
   moduleMocker: ModuleMocker | null;
   getVmContext(): Context | null;
+  seed: number;
```

Comment:
I don't think we need to store this on the `environment`. I see we for some reason do not pass `globalConfig` in to `jest-runtime` - we should change that. For now I think just add it as the last argument. The ctor is getting messy, so we should clean it up for Jest 30. But for now, just adding an optional last arg should be fine.